app/src/androidTest/java/com/example/android/architecture/blueprints/todoapp/HiltExt.kt

```diff
@@ -45,7 +44,10 @@ inline fun <reified T : Fragment> launchFragmentInHiltContainer(
             ApplicationProvider.getApplicationContext(),
             HiltTestActivity::class.java
         )
-    ).putExtra(EmptyFragmentActivity.THEME_EXTRAS_BUNDLE_KEY, themeResId)
+    ).putExtra(
+        "androidx.fragment.app.testing.FragmentScenario.EmptyFragmentActivity.THEME_EXTRAS_BUNDLE_KEY",
```

Comment:
If this is a string now, I guess the key itself can be removed? (it's a weird change)

Reply:
The new versions of the library made `EmptyFragmentActivity` not accessible anymore. We could either remove or leave it as is. I think it can be important for taking screenshots with the right theme.